### DISCUSSION TITLE
HP Reverb Support

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/MixedRealityToolkit-Unity#10448] [Q&A] [answered]
Hi!

I was reading the documentation for MRTK 2.6.1 (https://docs.microsoft.com/en-us/windows/mixed-reality/mrtk-unity/release-notes/mrtk-26-release-notes?view=mrtkunity-2021-05) and saw "HP Motion Controllers now supported with MRTK".

Before I purchase the hardware, I would like to confirm if HP motion controllers work with HoloLens 2? And how is the tracking compared to the built-in hand tracking..

**Comment by keveleigh** [comment, marked as answer]
To clarify, these controllers don't work with HoloLens 2. They're intended for use with the WMR VR headsets https://docs.microsoft.com/windows/mixed-reality/enthusiast-guide/vr-journey
